contents/blog/measure-product-market-fit.md

```diff
@@ -0,0 +1,394 @@
+
+> 💡 **PostHog Tip:** You can create your own user engagement metric in PostHog by creating an [Action](/docs/data/actions). Actions enable you to combine individual events (e.g. editing a page, sending a message, etc.) and track them as one metric. You can use Actions in numerous insight types, including trends, funnels, and retention insights.
+
+### Who should track user engagement?
+
+Everyone. It's basically impossible to measure product-market fit without it. The real challenge is tracking the right things – i.e. avoiding vanity metrics. See our guide to the [most useful B2B product metrics](/blog/b2b-saas-product-metrics) for help here. 
+
+| **Pros**                                                                                                                 | **Cons**                                                                                                                 |
+|--------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------| 
+| <span className="text-green text-lg">✔</span> Tracks the real value users gain from your product                         | <span className="text-red text-lg">✖</span> High engagement isn't necessarily predictive of revenue                      |
+| <span className="text-green text-lg">✔</span> Passive to maintain – set it up once and track forever                     | <span className="text-red text-lg">✖</span> Can provide a false positive if you're tracking the wrong things             |
+| <span className="text-green text-lg">✔</span> Easy to understand for all teams                                           | <span className="text-red text-lg">✖</span> Requires user feedback or session replays to understand behavior             |
+| <span className="text-green text-lg">✔</span> Can be broken down by user cohorts for comparison                          |           |
+| <span className="text-green text-lg">✔</span> You can A/B test ways to improve user engagement                           |           |
+
+## Indicator #4: Paying ICP customers
+
+- **Type:** Leading indicator
+- **Good for:** B2B SaaS products and subscription services
+
+It's tempting to treat revenue as a measure of product-market fit, but it's imperfect. It's easily skewed by one or two big contracts, or "design partners" who guide your product to solve their very specific needs.
+
+Instead, create an [Ideal Customer Profile](/blog/creating-ideal-customer-profile) (ICP) and track growth among paying customers who fit your ICP. Your ICP should be specific, and include needs, haves, and common attributes. 
+
+Here's PostHog's first ICP:
+
+![posthog icp](../images/blog/pmf-game-guide/posthog-icp.png)
+
+How much your ICP customers are paying is _less important_ than _how many_ ICP customers you have, and if they're growing in number.
```

Comment:
I don't fully buy this argument. If you have a couple big customers in your ICP with long sales cycles, could still be a good sign?

Reply:
Agree with Ian here. It's very case by case and there'd be many caveats to explain it fully. Best to just say that teams need to form their own opinion on this balance IMHO.